src/transformState.js

```diff
@@ -46,7 +46,25 @@
  * @param {Object[]} transforms
  */
 function transformState(state, transforms) {
-  // write code here
+  for (let i = 0; i < transforms.length; i++) {
+    if (transforms[i].operation === 'addProperties') {
+      for (const key in transforms[i].properties) {
+        state[key] = transforms[i].properties[key];
```

Comment:
Всі ```transforms[i]``` можна скоротити до ```i```:
```suggestion
        state[key] = i.properties[key];
```
якщо вище скористатися циклом for of.
Але я не наполягатиму